docs/internal_contributors.md

```diff
@@ -0,0 +1,70 @@
+# Internal i.AI contributors to ThemeFinder
+
+At present we are not accepting contributions to `themefinder` as it is undergoing rapid development within i.AI. If you have a suggestion, please raise an [issue](https://github.com/i-dot-ai/themefinder/issues), fill in our [feedback form](https://forms.gle/85xUSMvxGzSSKQ499), or email us at [packages@cabinetoffice.gov.uk](mailto:packages@cabinetoffice.gov.uk).
```

Comment:
Should we put this information in full in `CONTRIBUTING.md`?

Reply:
Yes - now updated: https://github.com/i-dot-ai/themefinder/pull/17/commits/b9921d06e28a778f8b1b00bfe4b961f26698bad8